lib/navigation/preprocess_encoders.hpp

```diff
@@ -15,11 +15,15 @@ class EncodersPreprocessor {
 
   void checkReliable(const core::EncoderData &encoder_data);
 
+  core::Float EncodersPreprocessor::calculating_median(const core::EncoderData encoder_data);
```

Comment:
You don't define class methods with `EncodersPreprocessor::`. You're already in the class. You only have to use that in the cpp file.